code/src/java/plugin/lsttokens/campaign/SaveToken.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2008 (C) Thomas Parker <thpr@users.sourceforge.net>
+ * 
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ * 
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ */
+package plugin.lsttokens.campaign;
+
+import pcgen.cdom.enumeration.ListKey;
+import pcgen.core.Campaign;
+import pcgen.persistence.lst.CampaignSourceEntry;
+import pcgen.rules.persistence.token.AbstractBasicCampaignToken;
+import pcgen.rules.persistence.token.CDOMPrimaryToken;
+
+/**
+ * Class deals with SAVES Token
```

Comment:
@LegacyKing  So the javadoc comment above refers to a SAVES token, as does the ListKey.FILE_SAVES but the tokenname is SAVE (no S). It is normally best to keep things exactly the same to avoid confusion unless there is a specific reason not to.